cmd/cql/internal/adapter.go

```diff
@@ -0,0 +1,71 @@
+package internal
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	"github.com/CovenantSQL/CovenantSQL/sqlchain/adapter"
+	"github.com/CovenantSQL/CovenantSQL/utils"
+)
+
+var (
+	adapterAddr string // adapter listen addr
+
+	adapterHTTPServer *http.Server
+)
+
+var CmdAdapter = &Command{
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var CmdAdapter should have comment or be unexported